packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -513,9 +517,9 @@ class RangeMaintainingScrollPhysics extends ScrollPhysics {
       maintainOverscroll = false;
       if (oldPosition.minScrollExtent.isFinite && oldPosition.maxScrollExtent.isFinite &&
           newPosition.minScrollExtent.isFinite && newPosition.maxScrollExtent.isFinite) {
-        // In addition, if the position changed then we only enforce
-        // the new boundary if the previous boundary was not entirely
-        // finite. A common case where the position changes while one
+        // In addition, if the position changed then we don't enforce
+        // the new boundary if both new and previous position are entirely finite.
```

Comment:
nit: position -> positions

Reply:
that said, positions are always finite. i'm not sure what this is intended to convey, can you take a look?

Reply:
I intended this to be a "boundary", changed it now